v22.2/create-schedule-for-backup.md

```diff
@@ -17,9 +17,17 @@ To use the other backup features, you need an [Enterprise license](enterprise-li
 
 ## Required privileges
 
+{% include {{ page.version.version }}/backups/updated-backup-privileges.md %}
+
+### Existing required privileges
+
 - Only members of the [`admin` role](security-reference/authorization.html#default-roles) can run `CREATE SCHEDULE FOR BACKUP`. By default, the `root` user belongs to the `admin` role.
```

Comment:
This is no longer true, with the new backup privilege model in 22,2, right? 

(Or is this section meant to indicate the "current privilege model" for 22.1 and older?)

Reply:
It is the existing privilege model yes. I have added some preface sentences to make this clear on the support and such to this section. Since it is supported in v22.2 and earlier, I have not changed to "current" because I think we really what the fine-grained access to be "current" if that makes sense.